### PR TITLE
libgit2 GCCcore-10.2.0

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/l/libgit2/libgit2-1.1.0-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    ('binutils', '2.35'),
+    ('CMake', '3.18.4'),
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('PCRE', '8.44'),
+]
+
+osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
```
Comment (Micket) [inline easybuild/easyconfigs/l/libgit2/libgit2-1.1.0-GCCcore-10.2.0.eb:28]:
```suggestion
osdependencies = [OS_PKG_OPENSSL_DEV]
```

File: easybuild/easyconfigs/l/libgit2/libgit2-1.1.0-GCCcore-10.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+checksums = ['41a6d5d740fd608674c7db8685685f45535323e73e784062cf000a633d420d1e']
+
+builddependencies = [
+    ('binutils', '2.35'),
+    ('CMake', '3.18.4'),
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('PCRE', '8.44'),
```
Comment (Micket) [inline easybuild/easyconfigs/l/libgit2/libgit2-1.1.0-GCCcore-10.2.0.eb:25]: Please include in the dependencies list:
```
     # ('OpenSSL', '1.1.1h')
     # OS dependency should be preferred if the os version is more recent than this version
```

**Review by Micket — CHANGES_REQUESTED** [review]

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/338047781c61011d36e3d91e602d70d1 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @fizwit!

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Pull request merged**